elements/rh-accordion/rh-accordion-header.ts

```diff
@@ -0,0 +1,48 @@
+import type { ColorTheme } from '../../lib/context/color.js';
+
+import { customElement, property } from 'lit/decorators.js';
+
+import styles from './rh-accordion-header.css';
+import { BaseAccordionHeader } from '@patternfly/pfe-accordion/BaseAccordionHeader.js';
+
+import { colorContextConsumer } from '@patternfly/pfe-core/decorators.js';
```

Comment:
```Suggestion
import type { ColorTheme  } from '../../lib/context/color.js';

import { customElement, property } from 'lit/decorators.js';

import styles from './rh-accordion-header.css';
import { BaseAccordionHeader } from '@patternfly/pfe-accordion/BaseAccordionHeader.js';

import { colorContextConsumer  } from '../../lib/context/color.js';
```

Reply:
Great points @zeroedin, updated the header & demo!